src/util/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const API_ENDPOINT = "https://developersdo.github.io/opensource/data";
+
+export default {
+  getDevelopers() {
+    return axios.get(`${API_ENDPOINT}/users.json`).then(({ data }) => data);
+  },
+
+  getRepositories() {
+    return axios.get(`${API_ENDPOINT}/repos.json`).then(({ data }) => data);
+  }
```

Comment:
You can define a base URL like this.
```js
axios.defaults.baseURL = "https://developersdo.github.io/opensource/data";
```
Now you can do this:
```js
axios.get("/users.json").then(() => ...)
```

Reply:
Done